iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -51,7 +51,7 @@ class ConversationListViewController: ViewController, ConversationListCollection
     }
 
     override var canBecomeFirstResponder: Bool {
-        return self.presentedViewController.isNil
+        return self.presentedViewController.isNil && ConversationsManager.shared.activeConversation.exists
```

Comment:
We never want to show the accessory view if there is no active conversation. 

Correct me if I'm wrong.